Makefile

```diff
@@ -3,6 +3,10 @@ VERSION ?= 1.1.1
 # Default bundle image tag
 IMAGE_TAG_BASE ?= controller
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+# Default Namespace
+NAMESPACE ?= openshift-storage
+# Default Replaces
+REPLACES ?= 1.1.0
```

Comment:
Why do we need to specify REPLACES? can't we just calc that from the version?

Reply:
We can calculate but not in every scenario. Let's suppose the current version is v1.1.0 then how can we determine the z-stream of the previous version? It can be v1.0.9 or v1.0.1 or anything else.
What do you think?

Reply:
Ok, agreed. But can we at least calc the `skipRange` value from this value?

Reply:
skipRange is calculated depending on VERSION in the Makefile. So it would be `olm.skipRange: ">=0.0.1 <$(VERSION)`.